Scrap 7 database conecction

joaoomagico opened the pull request:
added Sql database class (to be re-writen)

joaoomagico committed:
new test file

joaoomagico committed:
database.py pure SQL, re-write with SQLAlchemy.

joaoomagico committed:
deleted test file.

joaoomagico committed:
added comment

joaoomagico committed:
db conecction + example try

jravolio commented on db_conecction.py:
```diff
@@ -0,0 +1,45 @@
+from sqlalchemy.orm import sessionmaker, declarative_base
+from sqlalchemy import Column, String, Date
+from datetime import date
+#would be better to create a new file just for the imports? 
```
No, all the imports used on the file should remain on it

jravolio commented on db_conecction.py:
```diff
@@ -0,0 +1,45 @@
+    session.rollback()
+    print("Error inserting post:", e)
+finally:
+    session.close()
```
the filename.py is good! keep this name and move to a folder called utils, also add a gitignore as in the main branch to your code

jravolio reviewed (changes requested):
Request changes, summary:

- create the DB class so we can interact with it
- Pull code from main
- create folder called utils for db managemente
- create file for models of the post
- don't commit comments that will not help in any way the understatement of the function

_**DONT USE THIS MUCH OF THE COPILOT**_

joaoomagico committed:
- deleted unused files
- added id to the table model
- cleaned db_conecction
- added add_post, get_post and delete_post functions

joaoomagico committed:
- Added example usage of functions
